middleware/timeout_test.go

```diff
@@ -0,0 +1,90 @@
+// +build go1.13
+
+package middleware
+
+import (
+	"errors"
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestTimeout(t *testing.T) {
+	e := echo.New()
+
+	t.Run("Skipper", func(t *testing.T) {
```

Comment:
Maybe this is just a matter of taste, but I prefer a separate test function for every test. Also it is more aligned with the other test implementations on Echo.

Reply:
Will do